src/tests/class_tests/openms/source/FASTAFile_test.cpp

```diff
@@ -180,15 +216,67 @@ START_SECTION([EXTRA] test_strange_symbols_in_sequence)
   data.push_back(temp_entry);
   data.push_back(temp_entry); // twice
 
+
   file.store(tmp_filename, data);
   file.load(tmp_filename, data2);
-  
+
   ABORT_IF(data2.size() != 2);
   TEST_EQUAL(data2[0] == temp_entry, true);
   TEST_EQUAL(data2[1] == temp_entry, true);
-
 END_SECTION
 
+
+START_SECTION(test_white_spaces)
+//test if spaces and tabulators are removed correctly
+    String tmp_filename;
+    NEW_TMP_FILE(tmp_filename);
+    FASTAFile file;
+    vector<FASTAFile::FASTAEntry> data, data2;
+
+                FASTAFile::FASTAEntry temp_entry;
+                temp_entry.identifier = String("P68509|1433F_BOVIN");
+                temp_entry.description = String("This is the description of the first protein");
+                temp_entry.sequence = String("GDREQLLQRAR LAEQ\tAERYDDMASAMKAVTEL");
+                data.push_back(temp_entry);
+
+                file.store(tmp_filename, data);
+                file.load(tmp_filename, data2);
+
+                ABORT_IF(data2.size() != 1);
+                TEST_EQUAL(data2[0].sequence == string("GDREQLLQRARLAEQAERYDDMASAMKAVTEL"), true);
+
+        END_SECTION
+
+        START_SECTION([EXTRA] test_position) //
+                // test if setPosition() works correctly
+                String tmp_filename;
+                NEW_TMP_FILE(tmp_filename);
+                FASTAFile file;
+                //vector<FASTAFile::FASTAEntry> data;
+                vector<pair<streampos, FASTAFile::FASTAEntry>> data;
+                FASTAFile::FASTAEntry temp_entry;
+                file.readStart(OPENMS_GET_TEST_DATA_PATH("FASTAFile_test.fasta"));
+
+                unsigned i =1;
+                while(i<5){ //read the first, say, 4 entries
+                file.readNext(temp_entry);
+                data.push_back(std::make_pair(file.position(), temp_entry));//remember their FASTAEntry's as well as their position.
+                i++;}
+                file.setPosition(data[1].first); //reset the position to the, say, 2nd entry
+                while(i<8){ //read the next 3 entries
+                    file.readNext(temp_entry);
+                    data.push_back(std::make_pair(file.position(), temp_entry));//and also remember their positions
+                    i++;
+                }//
+
+                ABORT_IF(data.size() != 7 || data[1].second.sequence != data[4].second.sequence || data[6].first != data[3].first );
```

Comment:
and compare all the entries which should match (by position, id and sequence) .. not just some arbitrary ones?